requirements.txt

```diff
@@ -41,7 +42,8 @@ pyflakes==1.6.0           # via flake8
 pygments==2.2.0           # via sphinx
 pymongo==3.5.1            # via mongoengine
 pyrabbit2==1.0.3
-pytest==3.4.2
+pytest==3.5.0
+pytest-mock==1.9.0
```

Comment:
Should pytest-mock be added to the requirements.in?

Reply:
Arg. Good catch.